### PR TITLE
[pritunl] add mac_addresses parameter

### PR DESCRIPTION
**Pull request by vvatlin**
##### SUMMARY

Pritunl VPN supports to limit user by his MAC addresses, added this parameter to the module

##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME

pritunl

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: changelogs/fragments/4535-pritunl-add-mac_addresses-parameter.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+---
+minor_changes:
+  - pritunl_user - add ``mac_addresses`` parameter
```
Comment (felixfontein) [inline changelogs/fragments/4535-pritunl-add-mac_addresses-parameter.yml:3]:
```suggestion
  - pritunl_user - add ``mac_addresses`` parameter (https://github.com/ansible-collections/community.general/pull/4535).
```

File: plugins/modules/net_tools/pritunl/pritunl_user.py [hunk 2]
```diff
@@ -82,28 +82,38 @@
...
         description:
             - Enable/Disable Gravatar usage for the user I(user_name).
+
+    user_mac_addresses:
+        type: list
+        elements: str
+        required: false
+        default: null
+        description:
+            - Allowed MAC Addresses for the user I(user_name).
```
Comment (felixfontein) [inline plugins/modules/net_tools/pritunl/pritunl_user.py:92]:
```suggestion
            - Allowed MAC addresses for the user I(user_name).
        version_added: 4.8.0
```

**Review by felixfontein — COMMENTED** [review]
Thanks for your contribution! I've added some first comments.

**Comment by vvatlin** [comment]
Sorry for delay, I'll test today.

File: plugins/modules/net_tools/pritunl/pritunl_user.py [hunk 3]
```diff
@@ -82,28 +82,37 @@
         default: null
         description:
             - Enable/Disable Gravatar usage for the user I(user_name).
+
+    user_mac_addresses:
+        type: list
+        elements: str
+        description:
+            - Allowed MAC addresses for the user I(user_name).
+        version_added: 4.8.0
```
Comment (felixfontein) [inline plugins/modules/net_tools/pritunl/pritunl_user.py:91]:
```suggestion
        version_added: 5.0.0
```

**Comment by vvatlin** [comment]
@felixfontein yes you are right, I've added the check to the 218 also.

File: plugins/modules/net_tools/pritunl/pritunl_user.py [hunk 4]
```diff
@@ -205,7 +215,7 @@ def add_or_update_pritunl_user(module):
                 user_params[key] = users[0][key]
 
             # 'groups' is a list comparison
```
Comment (felixfontein) [inline plugins/modules/net_tools/pritunl/pritunl_user.py:217]:
```suggestion
            # 'groups' and 'mac_addresses' are list comparison
```

**Comment by felixfontein** [comment]
Looks good to me.

@Lowess what do you think?

**Comment by felixfontein** [comment]
I'll merge beginning of next week if nobody complains.

**Comment by felixfontein** [comment]
@vvatlin thanks for your contribution!

**Pull request merged**